solidity/contracts/system/TBTCSystem.sol

```diff
@@ -638,6 +625,19 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         return false;
     }
 
+    /// @notice Calculates bond requirement in wei for the given lot size in
+    /// satoshis given the current BTCETH price.
```

Comment:
Super tiny nit that I'm not going to block merge on, but I would say “based on the current ETHBTC price”, mostly because we're using “given” earlier to refer specifically to something that is passed in.

Reply:
Wait wait, I can edit it 😆